### ISSUE TITLE
Add linkedNpc.addY

### ISSUE THREAD

**Issue by Drizz1** [FancyMcPlugins/FancyNpcs#63]
Add linkedNpc.addY

Because different npc entity types has different height and we should have ability to add y to hologram location. We can change manually Y in config, but then after hologram movehere it is broken.

**Comment by Drizz1** [comment]
Example Hologram: 

item: Diamond
scale: 5

linked to npc

**Comment by OliverSchlueter** [comment]
Yeah currently I'm aligning it to the eye-height, I'll do something about that
